script/browserify.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
Oh, no problem. I am happy to add the feature if you find it useful for your project. I like the way you do it in that `js_uml` library, so feel free to do it that way here too if you want!